miner/algo_common.go

```diff
@@ -551,14 +588,22 @@ func (envDiff *environmentDiff) commitSBundleInner(b *types.SBundle, chData chai
 
 		if el.Tx != nil {
 			receipt, _, err := envDiff.commitTx(el.Tx, chData)
+
 			if err != nil {
+				// if drop enabled, and revertible tx has error on commit,
+				// we skip the transaction and continue with next one
+				if algoConf.DropRevertibleTxOnErr && el.CanRevert {
+					log.Warn("Found error on commit for revertible tx, but discard on err is enabled so skipping.",
```

Comment:
Suggestion: Warn -> Info since it's expected behaviour (we do want to log it tho I agree)

Reply:
updated